student.rb

```diff
@@ -1,15 +1,14 @@
-class Student
-  attr_reader :age, :phase
-  attr_accessor :name
+require_relative 'person'
+require_relative 'highfiveable'
 
+class Student < Person
+
+  include Highfiveable
+  attr_reader :phase
```

Comment:
Every single class includes these! So just stick them in `Person`. (both the `Highfiveable` and the phase attr.